src/main/java/com/denizenscript/denizencore/objects/core/ElementTag.java

```diff
@@ -829,6 +855,9 @@ public static void registerTags() {
         // @group element checking
         // @description
         // Returns a number representing the difference between the two elements. (Uses Levenshtein logic).
+        // @example
+        // # Returns "2".
+        // - narrate <element[Hello world].difference[Helo wold]>

```

Comment:
Ideally an example here would seek to make clear how this tag is actually meant to be used, given that it's not necessarily an obvious concept